Makefile

```diff
@@ -113,6 +116,7 @@ releaseall: RELEASE_ARGS := "-a 386 -a amd64 -a arm64 -a armel -a armhf -a ppc64
 releaseall: release
 
 .PHONY: release
+release: CONTAINER_ENGINE_BUILD_FLAGS := --build-arg GO_VERSION=1
```

Comment:
There can be a case where CI is not updated, but still the release happens with the latest versions. I dont think we should be in such a situation,also we had issues like https://github.com/opencontainers/runc/pull/4234  in the past.

Reply:
That being said, I do somewhat question how important it is if our release binaries are built with EOL'd Go versions -- we don't rebuild old release binaries, so every binary we release is destined to have been built by an EOL'd compiler (or an older release with a security issue) at some point.

I guess trying to go for the newest one from the start is the least bad option, but I do wonder how helpful this is in general.